src/main/java/org/folio/rest/impl/ProtectionHelper.java

```diff
@@ -199,8 +200,7 @@ private Future<List<String>> getOpenForReadAcqUnitIds() {
           .collect(Collectors.toList());
 
         if (logger.isDebugEnabled()) {
-          logger.debug("{} acq units with 'protectRead==false' are found: {}", ids.size(), StreamEx.of(ids)
-            .joining(", "));
+          logger.debug("{} acq units with 'protectRead==false' are found: {}", ids.size(), StringUtils.join(ids, ", "));
```

Comment:
starting name of method is recommended
 
logger.debug("getOpenForReadAcqUnitIds:: {} acq units with 'protectRead==false' are found: {}", ids.size(), StringUtils.join(ids, ", "));